pykern/pkcli/test.py

```diff
@@ -177,6 +185,8 @@ def _try(output, restartable):
                     output=output,
                     env=_env(restartable),
                 )
+                if _runtime_warning(output):
+                    raise AssertionError("coroutine not awaited, warning raised")
```

Comment:
I think what should happen is that we should append the FAILURE description to the end of output. We could actually edit the pytest output:

`======================== 3 passed, 2 warnings in 27.88s ========================`

We don't know what failed or passed here, but we can simply replace this line with something like:

```
FAILED: Detected asyncio problem: coroutine was never awaited
```